### PR TITLE
add connected source docs

### PR DESCRIPTION
**Pull request by colleenkerr**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+nav_title: Connected Sources
+article_title: Connected SOurces
```
Comment (andrewcarlberg) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:3]: Did you mean to have the capital O in Sources?

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+GRANT USAGE ON DATABASE BRAZE_CLOUD_PRODUCTION TO ROLE BRAZE_INGESTION_ROLE;
+GRANT USAGE ON SCHEMA BRAZE_CLOUD_PRODUCTION.INGESTION TO ROLE BRAZE_INGESTION_ROLE;
+
+-- grant access to all current and future tables or views in the schema
+GRANT SELECT ON ALL TABLES IN SCHEMA BRAZE_CLOUD_PRODUCTION.INGESTION TO ROLE BRAZE_INGESTION_ROLE;
+GRANT SELECT ON FUTURE TABLES IN SCHEMA BRAZE_CLOUD_PRODUCTION.INGESTION TO ROLE BRAZE_INGESTION_ROLE;
+
+-- grant access to specific tables or views in the schema
+GRANT SELECT ON TABLE BRAZE_CLOUD_PRODUCTION.INGESTION.USERS_ATTRIBUTES_SYNC TO ROLE BRAZE_INGESTION_ROLE;
+```
```
Comment (andrewcarlberg) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:60]: We need the ability to create tables as well.

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+## Product setup
+
+Cloud Data Ingestion connected sources require some setup on the Braze side and in your instance. A connected source may reference one or more tables, so the user created for Braze to use should have permissions for all tables you want to be available in the connected soure. Follow these steps to set up the integration:
+
+{% tabs %}
+{% tab Snowflake %}
+1. Create a new connected source in the Braze dashboard.
+2. Retrieve the public key provided in the Braze dashboard and [append it to the Snowflake user for authentication](https://docs.snowflake.com/en/user-guide/key-pair-auth.html).
+3. Test the integration
+4. Use the connected source to create one or more CDI Segmetns
```
Comment (julianw-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:34]: typo `Segmetns`

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+{% tabs %}
+{% tab Snowflake %}
+You may set up multiple integrations with Braze, but each integration should be configured to connect a different schema. When creating additional connections, you may reuse existing credentials if connecting to the same Snowflake account.
+
+If you reuse the same user and role across integrations, you will **not** need to go through the step of adding the public key again.
+{% endtab %}
+{% endtabs %}
+
+## Using the connected source
+Once the source is created, it can be used to create one or more CDI Segments. For more information on creating a segment with this source, see the [CDI Segments documentation]({{site.baseurl}}/user_guide/engagement_tools/segments/segment_extension/cdi_segments/)).
```
Comment (julianw-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:162]: extra parenthesis at the end?

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+{% alert important %}
+Because connected sources run on your data warehouse directly, you will incur all costs associated with running these queries in your data warehouse. Connected sources do not consume data points, and CDI segments do not consume SQL Segment credits.
+{% endalert %}
+
+
+## Integration
+
+Cloud Data Ingestion connected sources require some setup on the Braze side and in your instance. 
+
+A connected source may reference one or more tables, so the user created for Braze to use should have permissions for all tables you want to be available in the connected source. Follow these steps to set up the integration:
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:29]: I'm a little confused how these numbered steps relate, if at all, to the steps below. Are they supposed to be a separate prerequisite that you need to accomplish before you can begin setting up connected sources? Or are these a different way to number the steps below?

Comment (colleenkerr) [reply _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:29]: I pushed a commit to try and clarify. The goal of this is just to show a brief summary of what the setup process will require. Can remove if you think that's better!

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md (hunk 2 above)
Comment (colleenkerr) [reply _docs/_user_guide/data_and_analytics/cloud_ingestion/connected_sources.md:67]: Pushed a change to add this. Let me know if it looks ok!

**Review by josh-mccrowell-braze — APPROVED** [review]

**Pull request merged**